src/link.js

```diff
@@ -115,6 +115,7 @@ const Link = (
   const location = router.store.history
     .createLocation(locationDescriptor);
 
+
```

Comment:
Nope, I just forgot to turn off "Ensure Single Trailing Newline" in Atom on this machine 😭